src/greatwall/gui.py

```diff
@@ -183,35 +367,34 @@ def __init__(self):
         # List of widgets lists
         self.state_widgets: list[list[QWidget]] = []
 
-        # Threaded execution objects
-        self.worker_thread = GreatWallWorker(self.greatwall)
-
         # Launch UI
         self.main_states: list[QState()] = []
         self.error_states: list[QState()] = []
         self.main_gui_sm = QStateMachine()
         self.loop_dynamic_sm = QStateMachine()
         self.dynamic_states: list[QState()] = []
         self.init_ui()
-        self.init_main_gui_sm()
+        self.init_main_app_state()
 
     def init_ui(self):
         self.setWindowTitle("Great Wall Sample")
-        self.setGeometry(100, 100, 100, 100)
+        self.setGeometry(100, 100, 500, 500)  # left, top, width, height
 
-        self.password_text.setGeometry(0, 0, 100, 50)
         # Hardcode to fast tests
-        # self.password_text.setText("viboniboasmofiasbrchsprorirerugugucavehistmiinciwibowifltuor")
```

Comment:
Why commented the hardcore default test?

Reply:
This line will add a password to the password box text at starting of app, we use this only to fastly test our imlementation instead of rewrite the password. Although, this is not the professianal way to test app, but it's helping us to make sure we have a working example that we can use in launching.